bentoml/_internal/io_descriptors/multipart.py

```diff
@@ -44,28 +59,30 @@ def predict(img, annotations):
     curl -i -F img=@test.jpg -F annotations=@test.json localhost:5000/predict
     """
 
-    def __init__(self, **inputs: IODescriptor[_DescriptorType]):
+    def __init__(self, **inputs: _DescriptorType):
         for descriptor in inputs.values():
-            if isinstance(descriptor, Multipart):
+            if isinstance(descriptor, Multipart):  # pragma: no cover
                 raise InvalidArgument(
                     "Multipart IO can not contain nested Multipart item"
                 )
 
-        self._inputs = inputs
+        self._inputs = inputs  # type: t.Dict[str, _DescriptorType]
```

Comment:
can you use `self._inputs: TYPE = inputs` here?

Reply:
sure I will add it tmr morning